lib/lib_graph.go

```diff
@@ -107,6 +106,20 @@ func BuildSynchronizedCompleteGraph(n int) ([]Node, Synchronizer) {
 	return vertices, synchronizer
 }
 
+func BuildSynchronizedCompleteGraphWithLoopsAndOrderedNeighbours(n int) ([]Node, Synchronizer) {
```

Comment:
Please just add two bool flags to BuildSynchronizedCompleteGraph: one for loops, one for orientation (see BuildSynchronizedHypercube for reference) and use that function instead.